djmoney/contrib/exchange/apps.py

```diff
@@ -0,0 +1,3 @@
+class ExchangeConfig(AppConfig):
```

Comment:
Looks like you're missing an import:
`from django.apps import AppConfig`

Reply:
Added!